core/editor/src/tools/ellipse.rs

```diff
@@ -0,0 +1,16 @@
+use crate::events::Event;
+use crate::tools::Tool;
+
+pub struct Ellipse;
+
+impl Tool for Ellipse {
+	fn handle_input(&mut self, event: Event) {
+		todo!();
+	}
+}
+
+impl Default for Ellipse {
+	fn default() -> Self {
+		Self
+	}
+}
```

Comment:
I suppose you could just #[derive(Default)] it for now

Reply:
I forgot that `Default` is derivable, I will fix that.